plugins/dominant-color-images/readme.txt

```diff
@@ -1,15 +1,15 @@
-=== Dominant Color Images ===
+=== Image Placeholders ===
 
 Contributors:      wordpressdotorg
 Requires at least: 6.4
 Tested up to:      6.5
 Requires PHP:      7.0
-Stable tag:        1.0.2
+Stable tag:        1.1.0
 License:           GPLv2 or later
 License URI:       https://www.gnu.org/licenses/gpl-2.0.html
 Tags:              performance, images, dominant color
 
-Adds support to store the dominant color of newly uploaded images and create a placeholder background of that color.
+Displays placeholders based on an image's dominant color while the image is loading.
 
 == Description ==
```

Comment:
Perhaps the first line of the description should say:

> _This plugin was formerly known as Dominant Color Images._

Reply:
Sounds good

Reply:
Done in https://github.com/WordPress/performance/pull/1101/commits/0c04a92f8cdf1eb4067a0ca11f2c0b92ee4a7b24